chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -108,9 +110,14 @@
       group node[:neutron][:platform][:group]
     end
   end
+when "vmware"
```

Comment:
> why?

Replaced with an ```if``` in https://github.com/crowbar/barclamp-neutron/commit/6573fd00800313318448834fb6177af75676afa8